tests/php82_metadata.phpt

```diff
@@ -0,0 +1,48 @@
+--TEST--
+Dynamic property support in php 8.2+
+--SKIPIF--
+<?php if (!class_exists('AllowDynamicProperties') || PHP_VERSION_ID >= 90000) die('skip PHP >=8.2 && < 9.0 only'); ?>
```

Comment:
Why the `< 9` check?

Reply:
You're right, it's unnecessary. Removed.